notNeededPackages.json

```diff
@@ -121,6 +121,12 @@
             "asOfVersion": "3.4.0"
         },
         {
+            "libraryName": "electron",
+            "typingsPackageName": "electron",
+            "sourceRepoURL": "https://github.com/electron/electron",
+            "asOfVersion": "2.0.0"
```

Comment:
Updated.
The doc(https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) said:
> asOfVersion: A stub will be published to @types/foo with this version. Should be higher than any currently published version.

The current `@types/electron` is `v1.4.38`, so I thought `>v1.4.38` should be OK, then I thought it is   break change for `@types/electron`, so it should be `2.0.0`.

I suggest the doc about `asOfVersion` be clearer,  to avoid misunderstanding.